tests/base.py

```diff
@@ -153,27 +177,32 @@ def run_and_verify_check_mode(self, conn_id):
         This should be ran prior to field selection and initial sync.
         Return the connection id and found catalogs from menagerie.
         """
-        # run in check mode
+        # Run in check mode
         check_job_name = runner.run_check_mode(self, conn_id)
 
-        # verify check exit codes
+        # Verify check exit codes
         exit_status = menagerie.get_exit_status(conn_id, check_job_name)
         menagerie.verify_check_exit_status(self, exit_status, check_job_name)
 
         found_catalogs = menagerie.get_catalogs(conn_id)
-        self.assertGreater(len(found_catalogs), 0, msg="unable to locate schemas for connection {}".format(conn_id))
+        self.assertGreater(len(found_catalogs), 0,
+                           msg="unable to locate schemas for connection {}".format(conn_id))
```

Comment:
Instead of using format, start using f'string if you're using python >= 3.6 for the respective tap

Reply:
Replaced all occurrences of format() by f-strings